config/initializers/sabisu_rails.rb

```diff
@@ -0,0 +1,36 @@
+SabisuRails.setup do |config|
+
+  # Base uri for posting the
+  # config.base_api_uri = nil
+
+  # Ignored attributes for building the forms
+  # config.ignored_attributes = %w{ created_at updated_at id }
+
+  # HTTP methods
+  # config.http_methods = %w{ GET POST PUT DELETE PATCH }
+
+  # Headers to include on each request
+  #
+  # You can configure the api headers fairly easy by just adding the correct headers
```

Comment:
Line is too long. [84/80]<br>Incorrect indentation detected (column 2 instead of 3).